client/jetpack-connect/plans.jsx

```diff
@@ -69,7 +70,7 @@ class Plans extends Component {
 
 	maybeRedirect() {
 		if ( this.props.isAutomatedTransfer ) {
-			this.props.goBackToWpAdmin( this.props.selectedSite.URL + JETPACK_ADMIN_PATH );
+			externalRedirect( this.props.selectedSite.URL + JETPACK_ADMIN_PATH );
```

Comment:
I was testing this case and thinking about changing it to redirect to calypso plans. But looking back at the original [PR](https://github.com/Automattic/wp-calypso/pull/12695), it seems that this is for when AT sites have somehow become disconnected, and the remedy is to go through the jetpack connect procedure. So perhaps going to wpadmin does make sense in this context.

Suggest we add a comment such as `// When reconnecting broken AT sites`.

Does this make sense @artpi?